### DISCUSSION TITLE
Failed transsaction

### DISCUSSION THREAD

**Discussion by genius2091** [subquery/subql#1063] [Frontier TestNet - Bug Reporting]
![photo_2022-05-23 18 39 23](https://user-images.githubusercontent.com/73879972/169867426-0f4a032b-355d-4ca3-9fbb-6998baada5c7.jpeg)

**Comment by mzxyz** [comment]
Thanks reporting this issue, could you provide more context for this transaction, like what kind of action for it.
The error seems the problem with insufficient funds, could you make sure your signer has enough dev token, thanks
